sdk/keyvault/azure-keyvault-certificates-apis/azure/keyvault/certificates/_shared/_generated/v7_0/aio/operations_async/_key_vault_client_operations_async.pyi

```diff
@@ -0,0 +1,85 @@
+# Stubs for azure.keyvault.certificates._shared._generated.v7_0.aio.operations_async._key_vault_client_operations_async (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any, Optional
+
+class KeyVaultClientOperationsMixin:
+    async def create_key(self, vault_base_url: Any, key_name: Any, kty: Any, key_size: Optional[Any] = ..., key_ops: Optional[Any] = ..., key_attributes: Optional[Any] = ..., tags: Optional[Any] = ..., curve: Optional[Any] = ..., *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    async def import_key(self, vault_base_url: Any, key_name: Any, key: Any, hsm: Optional[Any] = ..., key_attributes: Optional[Any] = ..., tags: Optional[Any] = ..., *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    async def delete_key(self, vault_base_url: Any, key_name: Any, *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    async def update_key(self, vault_base_url: Any, key_name: Any, key_version: Any, key_ops: Optional[Any] = ..., key_attributes: Optional[Any] = ..., tags: Optional[Any] = ..., *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    async def get_key(self, vault_base_url: Any, key_name: Any, key_version: Any, *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    def get_key_versions(self, vault_base_url: Any, key_name: Any, maxresults: Optional[Any] = ..., *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    def get_keys(self, vault_base_url: Any, maxresults: Optional[Any] = ..., *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    async def backup_key(self, vault_base_url: Any, key_name: Any, *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    async def restore_key(self, vault_base_url: Any, key_bundle_backup: Any, *, cls: Optional[Any] = ..., **kwargs: Any): ...
+    async def encrypt(self, vault_base_url: Any, key_name: Any, key_version: Any, algorithm: Any, value: Any, *, cls: Optional[Any] = ..., **kwargs: Any): ...
```

Comment:
A thing I do a lot when learning a new API is I'll type 'key' (or something similar) to see all the APIs that manipulate keys. I wouldn't find this (or decrypt, sign, etc..). It seems we've been pretty consistent about putting key in the name of <op>_(key|secret) - would these fall under that same naming convention?